tests/testthat.R

```diff
@@ -1,4 +1,5 @@
 library(testthat)
+library(pharmaverseadam)
```

Comment:
I don't believe that this needs to be here

Reply:
Thanks @bms63. I've removed this line and pushed.